gridt/src/app/movements/movements.page.html

```diff
@@ -1,73 +1,77 @@
 <ion-header>
-  <ion-toolbar >
+  <ion-toolbar>
     <ion-buttons slot="start">
       <ion-menu-button></ion-menu-button>
     </ion-buttons>
-    <ion-title>
-      Movements
-    </ion-title>
-    <ion-buttons slot="primary">
-      <ion-button routerLink="/add">
-        <ion-icon name="add" slot="icon-only"></ion-icon>
-      </ion-button>
-    </ion-buttons>
+    <ion-title>Movements </ion-title>
   </ion-toolbar>
 </ion-header>
 
-
 <ion-content padding>
   <!-- First an explicit search bar -->
   <ion-row>
     <ion-col size-md="6" offset-md="3">
       <ion-toolbar>
-        <ion-searchbar 
-          showCancelButton="focus" 
+        <ion-searchbar
+          showCancelButton="focus"
           [(ngModel)]="searchText"
         ></ion-searchbar>
       </ion-toolbar>
     </ion-col>
   </ion-row>
-<!-- Then print a list of all the movements -->
-<ion-list>
-    <ion-grid
-      *ngIf="!(movements$ | async)">
-    <ion-row>
-      <ion-col size="12" size-sm="8" offset-sm="2" text-center>
-        <p>There are no movements  right now, please come back later!</p>
-      </ion-col>
-    </ion-row>
-  </ion-grid>
+  <!-- Then print a list of all the movements -->
+  <ion-list>
+    <ion-grid *ngIf="!(movements$ | async)">
+      <ion-row>
+        <ion-col size="12" size-sm="8" offset-sm="2" text-center>
+          <p>There are no movements right now, please come back later!</p>
+        </ion-col>
+      </ion-row>
+    </ion-grid>
     <ion-grid *ngIf="movements$ | async">
-
-  <!-- Create a list item for each search result.
+      <!-- Create a list item for each search result.
```

Comment:
Thank you for looking out for code quality, much appreciated! I personally don't prefer the way this comment got aligned.